src/ec/packages/ec-component-footer-core/ecl-footer-core.html.twig

```diff
@@ -40,49 +40,56 @@
 
 <footer class="{{ _css_class }}"{{ _extra_attributes|raw }}>
   <div class="ecl-container ecl-footer-core__container">
-  {% if _sections is not empty and _sections is iterable %}
-    {% for key, section in _sections %}
-      {% set _section_id = key + 1 %}
-      {% set _section_class = 'ecl-footer-core__section ecl-footer-core__section' ~ _section_id %}
-      {% if section.section_class_name is defined and section.section_class_name is not empty %}
-        {% set _section_class = _section_class ~ ' ' ~ section.section_class_name %}
+{% if _sections is not empty and _sections is iterable %}
+  {% for key, section in _sections %}
+    {% set _section_id = key + 1 %}
+    {% set _section_class = 'ecl-footer-core__section ecl-footer-core__section' ~ _section_id %}
+    {% if section.section_class_name is defined and section.section_class_name is not empty %}
+      {% set _section_class = _section_class ~ ' ' ~ section.section_class_name %}
+    {% endif %}
+    <section class="{{ _section_class }}">
+    {# content before #}
+    {% if section.content_before is defined and section.content_before is not empty %}
+      <div class="ecl-footer-core__content">{{ section.content_before|raw }}</div>
+    {% endif %}
+    {# title #}
+    {% if section.title is defined and section.title is not empty %}
```

Comment:
Why are we declaren first local variable _sections where we afterwards are going to use section in the for loop. Doesn't it make more sense to decare this in the for loop aswell and make it {% for key, _section in _sections %} ?

Reply:
Ok, yes, make sense.